.gitignore

```diff
@@ -76,5 +76,6 @@ sftp-settings.json
 /bootstrap/cache/
 
 /resources/assets/js/messages.js
-
-composer-installer
+
+composer-installer
+prepros.cfg
```

Comment:
you probably should check [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Various other editor-specific files in here should also be removed as well but at least we shouldn't add a new one.